flair/embeddings.py

```diff
@@ -326,6 +326,84 @@ def extra_repr(self):
     def __str__(self):
         return self.name
 
+class ELMoTransformerEmbeddings(TokenEmbeddings):
+    """Contextual word embeddings using word-level Transformer-based LM, as proposed in Peters et al., 2018."""
+
+    def __init__(self, model_file: str):
+        super().__init__()
+
+        self.name = 'elmo-transformer'
+        self.static_embeddings = True
+        self.allen_nlp_utils = ELMoTransformerEmbeddings.AllenNlpUtils(model_file)
+
+        # embed a dummy sentence to determine embedding_length
+        dummy_sentence: Sentence = Sentence()
+        dummy_sentence.add_token(Token('hello'))
+        embedded_dummy = self.embed(dummy_sentence)
+        self.__embedding_length: int = len(embedded_dummy[0].get_token(1).get_embedding())
+
+    @property
+    def embedding_length(self) -> int:
+        return self.__embedding_length
+
+    class AllenNlpUtils:
+        def __init__(self, model_file):
+            from allennlp.modules.token_embedders.bidirectional_language_model_token_embedder import \
+                BidirectionalLanguageModelTokenEmbedder
+            from allennlp.data.token_indexers.elmo_indexer import ELMoTokenCharactersIndexer
+
+            self.lm_embedder = BidirectionalLanguageModelTokenEmbedder(
+                archive_file=model_file,
+                dropout=0.2,
+                bos_eos_tokens=["<S>", "</S>"],
```

Comment:
should be `bos_eos_tokens=("<S>", "</S>"),` instead. `bos_eos_tokens` expects Tuple, not List.

Reply:
Thanks, I've fix it in 102095f :)